Typescript development in Cirq and prototype bloch sphere implementation (#4171)
quantumlib/Cirq: cirq-web/README.rst

> 

Reviewer: nit: remove new lines on the top

Reviewer: We should also add the Cirq logo here. This is the description that gets published to PyPi.

`Cirq <https://quantumai.google/cirq>`__ is a Python library for writing, manipulating, and optimizing quantum
circuits and running them against quantum computers and simulators.

> This module is **cirq-web**, which allows users to take advantage of browser based 3d visualization tools

Reviewer: ```suggestion
This module is **cirq-web**, which allows users to take advantage of browser based 3D visualization tools
```

> This module is **cirq-web**, which allows users to take advantage of Typescript-based 3d visualization tools

Reviewer: 👍

> This module is **cirq-web**, which allows users to take advantage of Typescript-based 3d visualization tools

Author: Done

and features in Cirq. cirq-web also provides a development environment for contributors to create and add 
their own visualizations to the module.

Documentation
-------------

> All current documentation for cirq-web can be found in the README files located in this module's subdirectories.

Reviewer: ```suggestion
Documentation for cirq-web can be found in the README files located in this module's subdirectories.
```

Also, link to the repo.

> Below is a quick example of how to generate a portable 3d rendering of a bloch sphere using cirq-web:

Reviewer: ```suggestion
Below is a quick example of how to generate a portable 3D rendering of the Bloch sphere using cirq-web:
```

.. code-block:: python

    from cirq_web import BlochSphere
    sphere = BlochSphere()
    sphere.generate_html_file()

This will create the file in the current working directory. There are additional options to specify the

> output directory, open the visualization in a browser, etc. 

Reviewer: ```suggestion
output directory or to open the visualization in a browser for example. 
```

> You can also view and interact with a bloch sphere in a Colab or Jupyter notebook setting

Reviewer: ```suggestion
You can also view and interact with a Bloch sphere in a Colab or Jupyter notebook setting
```

with the following:

.. code-block:: python

    from cirq_web import BlochSphere
    sphere = BlochSphere()
    display(sphere)

> Note that you can pass a state vector into the :code:`BlochSphere()` constructor to view a particular

Author: My thought process is that since this is the PyPI readme, and in production users won't be accessing the typescript at all, its safe to provide examples using just the Python side.
